app/routes/index.ts

```diff
@@ -21,6 +23,12 @@ const handleUnhandledError: Middleware = (
 initializeDb();
 router.use('/user', user);
 router.use('/auth', auth);
+router.use('/pizza', AuthVerify, pizza.createPizza);
+router.use('/get-pizza', AuthVerify, pizza.getPizzas);
```

Comment:
You are not differentiating between post get and update API calls?
Any reason?

Reply:
yeah, doing that now